Detectors/TRD/calibration/workflow/trd-dcs-sim-workflow.cxx

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DCStestWorkflow/DCSRandomDataGeneratorSpec.h"
+#include "Framework/runDataProcessing.h"
+
+o2::framework::WorkflowSpec defineDataProcessing(o2::framework::ConfigContext const& configcontext)
+{
+  std::vector<o2::dcs::test::HintType> dphints;
+  // for TRD
+  // for test, we use less DPs that official ones
+  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"trd_gasCO2", 0, 50.});
```

Comment:
What art the values 0 and 50.? A comment might help... 
If one of them is a default value, it would be good to put something non-sensical (e.g. -999.).

Reply:
I'll put a comment. It means for simulations the provided value is in the range from 0 to 50